### DISCUSSION TITLE
Excees row quota

### DISCUSSION THREAD

**Discussion by Jefflai0315** [Budibase/budibase#4524] [Help]
![image](https://user-images.githubusercontent.com/79297365/154257861-98e1d66e-9b5c-4586-99e5-d0b30b014ffd.png)

Hi, I dont think I have ever created so many rows. Will I be able to delete some rows?

**Comment by Jefflai0315** [comment]
@mjashanks Hey, I still not able to create new row into my database. Not even in the data page. Now it does not show the error, but I am not able to save it. ( the pop up modal screen stays). Could I get some advise?

**Reply by mjashanks** [reply]
Sorry @Jefflai0315  - I missed this - will look into it.

**Reply by mjashanks** [reply]
Hey @Jefflai0315 can you send your budibase cloud domain to support@budibase.com so I can look at this. Thanks.

e.g. `https://your-company.budibase.app`

**Reply by Jefflai0315** [reply]
@mjashanks I have emailed it. Have you looked into it ? The issue still persists.

**Reply by mjashanks** [reply]
not yet - it was the weekend, we'll investigate today.

**Reply by Jefflai0315** [reply]
My bad, thank for the response!

**Reply by mjashanks** [reply]
should be fixed now, we found an issue with the row count. Thanks for this - we've reset your quota.
